packages/qwik/src/core/v2/client/vnode.ts

```diff
@@ -686,6 +725,28 @@ export const vnode_getVNodeForChildNode = (
   return child as ElementVNode;
 };
 
+// this is trying to find the vnode for the element using Breadth-First Search
+export const vnode_getVNodeForElement = (
```

Comment:
Why do you need this?

Reply:
I removed it. It was for finding the q:template element